qlog/frame.go

```diff
@@ -130,8 +130,7 @@ func marshalCryptoFrame(enc *gojay.Encoder, f *logging.CryptoFrame) {
 
 func marshalNewTokenFrame(enc *gojay.Encoder, f *logging.NewTokenFrame) {
 	enc.StringKey("frame_type", "new_token")
-	enc.IntKey("length", len(f.Token))
-	enc.StringKey("token", fmt.Sprintf("%x", f.Token))
+	enc.ObjectKey("token", &token{Raw: f.Token})
```

Comment:
how sensitive is the token, should it appear in logs?

Reply:
Not very sensitive. All you can do with is initiate a new connection with a lower risk of receiving a Retry.
Also, you could correlate two connections.